Lib/test/test_code.py

```diff
@@ -141,9 +141,7 @@
     ctypes = None
 from test.support import (cpython_only,
```

Comment:
It seems that there are too many imports in the current line. Maybe it can be split. See line 146.

This suggestion applies to all changes (maybe I don't like to import too much from the from statement, so it's better to use * directly

(In this way, this `()` can be removed. Yes, it's still)

Reply:
I think this is fine. I don't think the PR should do a broader import cleanup -- just undo the changes related `@suppress_immortalization`.